repos/system_upgrade/el7toel8/actors/ipuworkflowconfig/libraries/ipuworkflowconfig.py

```diff
@@ -60,5 +68,29 @@ def get_booted_kernel():
         )
 
 
-def get_target_version():
-    return os.getenv('LEAPP_DEVEL_TARGET_RELEASE', None) or CURRENT_TARGET_VERSION
+def get_target_version(flavour=LEAPP_UPGRADE_FLAVOUR_DEFAULT):
+    """
+    Returns the target version for the given `flavour` of upgrade. The default value for `flavour` is `default`.
+    In case the environment variable `LEAPP_DEVEL_TARGET_RELEASE` is set, the value of it will be returned.
+    """
```

Comment:
```suggestion
    """
    Returns the target version for the given `flavour` of upgrade. The default value for `flavour` is `default`.

    In case the environment variable `LEAPP_DEVEL_TARGET_RELEASE` is set, the value of it will be returned.
    """
```